Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN apt-get update && \
 
 COPY requirements_all.txt requirements_all.txt
 RUN pip3 install --no-cache-dir -r requirements_all.txt
+RUN pip3 install --no-cache-dir colorlog
```

Comment:
As the colorlog package does not update regularly, it would be better to move it up in the dockerfile so people can use the cached layer when downloading a new image. Probably put it above nmap tracker because it is pretty stable.